docs/cimpl6ToolingReference.md

```diff
@@ -498,16 +497,34 @@ For many users, especially those with experience in object-oriented modeling, th
 
 ### Data Dictionary Export
 
-The Data Dictionary is a Microsoft Excel file containing a simplified, flattened list of model `Elements` and value sets, extracted from the FHIR profiles. The spreadsheet has five tabs:
+The Data Dictionary is a Microsoft csv file containing a simplified, flattened list of model `Elements` and value sets, extracted from the FHIR profiles. The spreadsheet has five tabs:
 
-* The _Key_ tab contains a key to the columns in the other tabs,
-* The _Profiles_ tab contains a list of the primary profiles and their descriptions,
-* The _Data Elements_ tab contains a list of [**MustSupport**](https://www.hl7.org/fhir/conformance-rules.html#mustSupport) and required elements associated with those profiles,
-* The _Value Sets_ tab contains a list of value sets used in the IG, and their descriptions,
+* The _Profiles_ tab contains a list of the profiles, with descriptions, that are defined in the IG,
+* The _Data Elements_ tab contains a list of profiles and elements, as defined in the IG [Content Profile](#content-profile-file)
+* The _Value Sets_ tab contains a list of value sets associated with the profiles and data elements , and their descriptions,
```

Comment:
I think it could be valuable to provide a list of column names and descriptions for each of the four tabs. This will allow someone reading the documentation to know exactly what information to expect in the data dictionary.

Reply:
Mint will provide the suggested information, Carmela to add in appendix.